indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -0,0 +1,179 @@
+    const newList = _.sortBy(list, [sortBy]);
+    if (sortDirection === SortDirection.DESC) {
+      newList.reverse();
+    }
+    return newList;
+  }
+
+  // eslint-disable-next-line no-shadow
+  function _sort({sortBy, sortDirection}) {
+    setSortBy(sortBy);
+    setSortDirection(sortDirection);
+    setSortedList(_sortList({sortBy, sortDirection}));
+  }
+
+  function _cellRenderer(dataKey, rowIndex) {
+    return (
+      <>
+        {dataKey === 'status' && (
+          <StateIndicator index={rowIndex} state={sortedList[rowIndex][dataKey]} circular />
+        )}
+        {dataKey === 'assigned' && <Icon name="user outline" />}
+        <TooltipIfTruncated>
+          <div index={rowIndex} className="rowColumn_tooltip" role="gridcell">
+            {sortedList[rowIndex][dataKey]}
+          </div>
+        </TooltipIfTruncated>
+      </>
+    );
+  }
+
+  function _headerCheckboxState(data) {
+    data.checked
+      ? setChecked([...Array(sortedList.length).keys()].map(row => sortedList[row].id))
+      : setChecked([]);
+  }
+
+  function _rowCheckboxState(data) {
+    const newRow = sortedList[data.index].id;
+    checked.includes(newRow)
+      ? setChecked(old => old.filter(row => row !== newRow))
+      : setChecked(old => [...old, newRow]);
+  }
+
+  return (
+    <>
+      <ManagementPageSubTitle title={title} />
+      <ManagementPageBackButton url={dashboardURL({confId: eventId})} />
+      <Button icon color="blue">
+        <Icon name="setting" />
+        <Translate>Customize List</Translate>
+      </Button>
+      <Button content={Translate.string('Assign')} />
+      <Button content={Translate.string('Unassign')} />
+      <Button content={Translate.string('Set status')} />
+      <Button content={Translate.string('Download all files')} />
+      <Button color="blue" floated="right" content={Translate.string('Assign to myself')} />
+      <div styleName="editable-list">
+        <WindowScroller>
+          {({height, isScrolling, onChildScroll, scrollTop}) => (
+            <Table
+              autoHeight
+              width={1000}
+              styleName="table"
+              height={height}
+              isScrolling={isScrolling}
+              onScroll={onChildScroll}
+              headerHeight={30}
+              rowHeight={40}
+              sort={_sort}
+              sortBy={sortBy}
+              sortDirection={sortDirection}
+              rowCount={sortedList.length}
+              scrollTop={scrollTop}
+              rowGetter={({index}) => sortedList[index]}
+            >
+              <Column
+                disableSort
+                dataKey="checkbox"
+                width={30}
+                headerRenderer={() => (
+                  <Checkbox
+                    indeterminate={checked.length > 0 && checked.length < sortedList.length}
+                    checked={checked.length === sortedList.length}
+                    onChange={(e, data) => {
```

Comment:
I think we can avoid the brackets here:
```suggestion
                    onChange={(e, data) => _headerCheckboxState(data)}
```

Or even better, and you just have to ignore the event in the callback function (that way we don't need an anonymous wrapper).
```suggestion
                    onChange={_headerCheckboxState(data)}
```

Reply:
I need to pass the `data` to the function.

Reply:
It doesn't work if you use `onChange={_headerCheckboxState}` and then receive it as: ?
```
const _headerCheckboxState = (__, data) => {
    console.log(data); // ...
}
```